scripts/main.js

```diff
@@ -6,11 +6,211 @@ function coolMenu() {
+  linkToSource: 'https://github.com/fdezcaminero/figmaPortfolio2023',
+},
+{
+  name: 'Uber',
+  description: 'Well upon our way',
+  featuredImage: {
+    imgSrc: 'resources/yellowLaptop.svg',
+    imgAlt: 'Uber Logo',
+  },
+  technologies: {
+    tech1: 'HTML/CSS',
+    tech2: 'JavaScript',
+    tech3: 'Perl',
+  },
+  liveVersion: 'https://fdezcaminero.github.io/figmaPortfolio2023/',
+  linkToSource: 'https://github.com/fdezcaminero/figmaPortfolio2023',
+},
+{
+  name: 'Microverse',
+  description: 'Learn to code',
+  featuredImage: {
+    imgSrc: 'resources/grayLaptop.svg',
+    imgAlt: 'Microverse Logo',
+  },
+  technologies: {
+    tech1: 'HTML/CSS',
+    tech2: 'JavaScript',
+    tech3: 'C/C++',
+  },
+  liveVersion: 'https://fdezcaminero.github.io/figmaPortfolio2023/',
+  linkToSource: 'https://github.com/fdezcaminero/figmaPortfolio2023',
+},
+{
+  name: 'YouTube',
+  description: 'You Tube',
+  featuredImage: {
+    imgSrc: 'resources/pinkScreen.svg',
+    imgAlt: 'YouTube Logo',
+  },
+  technologies: {
+    tech1: 'HTML/CSS',
+    tech2: 'Go',
+    tech3: 'C/C++',
+  },
+  liveVersion: 'https://fdezcaminero.github.io/figmaPortfolio2023/',
+  linkToSource: 'https://github.com/fdezcaminero/figmaPortfolio2023',
+},
+];
 
-let projects = {
+function loadProjects() {
+  document.getElementById('ProjectName1').innerHTML = projects[0].name;
+  document.getElementById('1tech1').innerHTML = projects[0].technologies.tech1;
+  document.getElementById('1tech2').innerHTML = projects[0].technologies.tech2;
+  document.getElementById('1tech3').innerHTML = projects[0].technologies.tech3;
+  document.getElementById('projectImage1').src = projects[0].featuredImage.imgSrc;
+  document.getElementById('projectImage1').alt = projects[0].featuredImage.imgAlt;
 
-};
+  document.getElementById('ProjectName2').innerHTML = projects[1].name;
+  document.getElementById('2tech1').innerHTML = projects[1].technologies.tech1;
+  document.getElementById('2tech2').innerHTML = projects[1].technologies.tech2;
+  document.getElementById('2tech3').innerHTML = projects[1].technologies.tech3;
+  document.getElementById('projectImage2').src = projects[1].featuredImage.imgSrc;
+  document.getElementById('projectImage2').alt = projects[1].featuredImage.imgAlt;
 
+  document.getElementById('ProjectName3').innerHTML = projects[2].name;
+  document.getElementById('3tech1').innerHTML = projects[2].technologies.tech1;
+  document.getElementById('3tech2').innerHTML = projects[2].technologies.tech2;
+  document.getElementById('3tech3').innerHTML = projects[2].technologies.tech3;
+  document.getElementById('projectImage3').src = projects[2].featuredImage.imgSrc;
+  document.getElementById('projectImage3').alt = projects[2].featuredImage.imgAlt;
 
-// name, description, featured image, technologies, link to live version, link to source.
\ No newline at end of file
+  document.getElementById('ProjectName4').innerHTML = projects[3].name;
```

Comment:
- [ ] Good job so far implementing the changes suggested by the previous reviewer :+1: :+1: , Your UI looks great and nice. But there are some changes you need to make before moving to the next milestone.
- As suggested by the previous reviewer, This part of your code goes against the DRY principle, you need to have a function that generates the HTML content dynamically and then appends it to the page, This will make your code more readable and dry and also you will not need to write extra javascript anytime you add a new project to the array.
- Please look into this as it's part of the requirement for this milestone. You can look at this sample codebase to see how the HTML was generated [LINK](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-2a099ca3b81cdfe23a8dc8eb7677c597dbb0996f90476b3dd497e0bc36fb0f69R90-R116)